src/console_reporter.cc

```diff
@@ -152,21 +175,19 @@ void ConsoleReporter::PrintRunData(const Run& result) {
     printer(Out, COLOR_CYAN, "%10lld", result.iterations);
   }
 
-  for (auto& c : result.counters) {
-    const std::size_t cNameLen = std::max(std::string::size_type(10),
-                                          c.first.length());
-    auto const& s = HumanReadableNumber(c.second.value, c.second.oneK);
```

Comment:
Ah, hm, this is already a string (too much `auto`!).
But still, appending to it is wrong.